lib/internal/abort_controller.js

```diff
@@ -13,6 +13,7 @@ const {
   Symbol,
   SymbolToStringTag,
   WeakRef,
+  PromiseResolve,
```

Comment:
nit: could we keep alphabetical/ASCII order here?

Reply:
Done